modules/perception/onboard/component/trafficlights_perception_component.cc

```diff
@@ -933,13 +926,12 @@ bool TrafficLightsPerceptionComponent::TransformDebugMessage(
 
   if (lights.size() > 0) {
     camera::CarPose pose;
-    if (GetCarPose(frame->timestamp, &pose)) {
-      Eigen::Matrix4d cam_pose;
-      pose.GetCameraPose("front_6mm", &cam_pose);
-      light_debug->set_distance_to_stop_line(stopline_distance(cam_pose));
-    } else {
-      AERROR << "error occured in calc distance to stop line";
+    if (!GetCarPose(frame->timestamp, &pose)) {
+      AERROR << "Error occured in calculate distance to stop line.";
```

Comment:
you may forget to add "return" here

Reply:
We should not return here as virtualizer need to start even though.

Reply:
Are you sure this is not a bug? IMO it seems to should return failure when this error occurs.

Reply:
Don't just make decision in terms of the logic, instead run the virtualization environment to check the behavior.